### PR TITLE
Enhancement (ci): Add more usage in `build.sh`

### PR DESCRIPTION
**Pull request by leojonathanoh**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -3,50 +3,66 @@
...
+    echo "  NO_TEST                 Whether to skip testing the game image after a successful build"
+    echo "                          Type: bool, optional"
+    echo "                          Possible values: 'true', 'false'"
+    echo "  NO_PUSH                 Whether to skip pushing the docker image after a successful build"
+    echo "                          Type: bool, optional"
+    echo "                          Possible values: 'true', 'false'"
+    echo "  REGISTRY_GOLDSOURCE     Docker repository when APPID=90 (goldsource games)"
+    echo "                          Type: string"
+    echo "  REGISTRY_SOURCE         Docker repository when APPID is not '90' (source games)"
+    echo "                          Type: string"
```
Comment (leojonathanoh) [inline build.sh:48]: should be updated after #137

Comment (joeltimothyoh) [reply build.sh:48]: can do so now

File: build.sh [hunk 2]
```diff
@@ -3,61 +3,75 @@
...
+    echo "  PIPELINE=update GAME_UPDATE_COUNT=1 GAME_VERSION=1123 APPID=90 CLIENT_APPID=10 GAME=valve MOD= FIX_APPMANIFEST=true STEAM_LOGIN=true DOCKER_REGISTRY=docker.io/mynamespace/valve REGISTRY_USER=xxx REGISTRY_PASSWORD=xxx STEAM_USERNAME=xxx STEAM_PASSWORD=xxx ./build.sh"
     echo
     echo "  # Build srcds/csgo image and push"
-    echo "  PIPELINE=build GAME_VERSION=13840 APPID=740 CLIENT_APPID=730 GAME=csgo LATEST=true REGISTRY_SOURCE=docker.io/mynamespace/csgo REGISTRY_USER=xxx REGISTRY_PASSWORD=xxx ./build.sh"
+    echo "  PIPELINE=build GAME_VERSION=13840 APPID=740 CLIENT_APPID=730 GAME=csgo LATEST=true DOCKER_REGISTRY=docker.io/mynamespace/csgo REGISTRY_USER=xxx REGISTRY_PASSWORD=xxx ./build.sh"
     echo
     echo "  # Update srcds/csgo image and push"
-    echo "  PIPELINE=update GAME_UPDATE_COUNT=1 GAME_VERSION=13840 APPID=740 CLIENT_APPID=730 GAME=csgo REGISTRY_SOURCE=docker.io/mynamespace/csgo REGISTRY_USER=xxx REGISTRY_PASSWORD=xxx ./build.sh"
+    echo "  PIPELINE=update GAME_UPDATE_COUNT=1 GAME_VERSION=13841 APPID=740 CLIENT_APPID=730 GAME=csgo DOCKER_REGISTRY=docker.io/mynamespace/csgo REGISTRY_USER=xxx REGISTRY_PASSWORD=xxx ./build.sh"
```
Comment (joeltimothyoh) [inline build.sh:74]: shouldn't it be `DOCKER_REPOSITORY` instead of `DOCKER_REGISTRY`

**Review by joeltimothyoh — CHANGES_REQUESTED** [review]

File: build.sh (hunk 2 above)
Comment (leojonathanoh) [reply build.sh:74]: yes

**Review by joeltimothyoh — APPROVED** [review]

**Pull request merged**